spec/helpers/i18n_helper_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spec_helper'
+
+describe I18nHelper do
+  let(:user) { create(:user) }
+
+  context "as guest" do
+    before do
+      allow(helper).to receive(:spree_current_user) { nil }
```

Comment:
where is `helper` defined?

Reply:
Rspec/Rails magic

Reply:
I couldn't find it Rails guides. That's why I ask. Perhaps is something from RSpec.

Reply:
https://github.com/rspec/rspec-rails#helper-specs